markdowns/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: tutorial
+colab: https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb
+toc: True
+title: "Text-to-Image search pipeline with MultiModal Retriever"
+last_updated: 2022-11-01
+level: "intermediate"
+weight: 95
+description: Use a MultiModalRetriever to build a cross modal search pipeline.
+category: "QA"
+aliases: ['/tutorials/multimodal']
+---
+    
+
+**Level**: Intermediate
+
+**Time to complete**: 20 minutes
+
+**Prerequisites**: Prepare the Colab environment (see links below).
+
+**Nodes Used**: InMemoryDocumentStore, MultiModalRetriever
+
+**Goal**: After completing this tutorial, you will have learned about the MultiModalRetriever, and built a simple retrieval pipeline that search relevant images given a text query.
+
```

Comment:
Tutorial 1, there is a paragraph underneath the intro table that describes what happens in the tutorial. I think this would be a good place to explain a little bit more about the process of mixed modality search. More specifically, some mention of the fact that:
- an image is embedded by a transformer model
- the embedding is representative of this image's contents
- the query is embedded by the same model and the embedding exists in the same vector space
- we perform nearest neighbours search link in text to text retrieval

Reply:
I added the description, can you maybe read it and let me know your thoughts?